src/navigation/Routes.android.js

```diff
@@ -333,6 +334,11 @@ function BSNavigator() {
         name={Routes.SWAPS_PROMO_SHEET}
         options={bottomSheetPreset}
       />
+      <BSStack.Screen
+        component={NotificationsPromoSheet}
+        name={Routes.NOTIFICATIONS_PROMO_SHEET}
```

Comment:
wondering if we should set up the promo sheet a lil diff to just take a type instead of adding routes for all of these 🤔

Reply:
i concur

Reply:
By "type" here you mean a route? It'll still result in routes, right? Or are you saying a single promosheet route that varies in content based on params?

Idk, I would typically argue that keeping things obvious is better. For all intents and purposes, these are routes, and it's nice to have a central file of ALL routes. And I like that the sheets are basically dumb components that just render text using `PromoSheet`. These are nice simple abstractions, easy to use and refactor.

Might not be what you meant, but I usually would say we should avoid the "black box" architectures where you pass in a single prop and the component magically handles the rest. Usually ends up with bloated API surface area, adding variant props to account for other use cases that weren't considered up front.